packs/network-flow-devices/config.yml

```diff
@@ -0,0 +1,46 @@
+# Name of the pack (required)
+name: network-flow-devices-pack
+
+# Displayed in the UI
+title: Network Flow Devices Pack
+
+# Long-form description of the pack (required)
+description: >-
+  The Network Flow Devices pack provides dashboards and alerts designed to give you deeper insights into your flow data.
+  Use this pack together with New Relic's Network Performance Monitoring (NPM) feature to visualize how users are consuming 
+  your bandwidth. (This pack contains Custom Visualizations which require Full User permissions to access)
+
+
+# Displayed in search results and recommendations. Summarizes a packs functionality.
+summary: >-
+  Network Flow Devices from NPM give you valuable insights into how your network bandwidth is being used.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the pack (required)
+authors:
+  - Zack Mutchler
```

Comment:
Should the author be `Kentik`?

Reply:
it's a list, so it could be Kentik first and then Zack?

Reply:
for support, it would be New Relic as the primary; I'll update